hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/genconf/GenerateOzoneRequiredConfigurations.java

```diff
@@ -106,9 +109,19 @@ public static void generateConfigurations(String path) throws
 
     for (OzoneConfiguration.Property p : allProperties) {
       if (p.getTag() != null && p.getTag().contains("REQUIRED")) {
-        if(p.getName().equalsIgnoreCase(OzoneConfigKeys.OZONE_ENABLED)) {
+        if (p.getName().equalsIgnoreCase(OzoneConfigKeys.OZONE_ENABLED)) {
           p.setValue(String.valueOf(Boolean.TRUE));
+        } else if (p.getName().equalsIgnoreCase(
+            OzoneConfigKeys.OZONE_METADATA_DIRS)) {
+          p.setValue(System.getProperty(OzoneConsts.JAVA_TMP_DIR));
+        } else if (p.getName().equalsIgnoreCase(
```

Comment:
This will be for local one node cluster?
As when HA comes, this will not work.

Reply:
@bharatviswa504 Yes, this is for people who just want to try it out. This is not meant for production use. When HA comes in, we may have to change it accordingly so that this works :)

Reply:
yes, we don't even know what the HA configs are. So we wil have to file a follow up JIRA when HA is ready. For time being I am going on with commit, since 0.4.1 release will be correct.